frontends/ast/simplify.cc

```diff
@@ -509,7 +511,7 @@ static int get_max_offset(AstNode *node)
 	// get the width from the MS member in the struct
 	// as members are laid out from left to right in the packed wire
 	log_assert(node->type==AST_STRUCT || node->type==AST_UNION);
-	while (node->type != AST_STRUCT_ITEM) {
+	while (node->range_left < 0) {
```

Comment:
It seems to me that this condition is equivalent before and afterwards. I'm guessing it was related to the the sub-struct array changes. If so, can this be reverted for now?

Reply:
Reverted.